lib/member_page.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :birth_date do
+    date_from(
+      line_for('Datëlindja').match(/(\d\d\.\d\d\.\d\d\d\d)/) { Regexp.last_match(1) }.to_s.tidy
+    )
+  end
+
+  field :birth_place do
+    record_for('Vendlindja')
+  end
+
+  field :area do
+    record_for('Qarku')
+  end
+
+  # TODO: This isn't so easy to extract as the text varies
+  # field :commissions do
+  #   puts line_for('Komisioni').split(/(Komisioni)/).drop(1).join('').tidy
+  # end
+
+  private
+
+  def line_for(str)
+    cells.find { |line| /#{str}:/ =~ line }.to_s
+  end
+
+  def record_for(str)
+    line_for(str).split(str).last.to_s.delete('.').delete(':').tidy
+  end
+
+  def table
+    noko.xpath('//div[contains(@class,"post-content")]/table')
+  end
+
+  def headings
```

Comment:
The name of this method doesn't really convey what it actually _is_. I certainly wouldn't expect a `headings` method to return a series of Regexps!